error: 'utf-8' codec can't decode bytes

Getting those errors for a [batch](https://anidb.net/anime/14449/?gid=15782#eplist), but only episode files 8,12, give this error?
```
[INFO] Processing file "d:\[Judas] Hamefura - S01E08.mkv"
[SUCCESS] Generated ed2k link.
[ERROR] Failed to get file info: 'utf-8' codec can't decode bytes in position 1397-1398: invalid continuation byte
[INFO] Processing file "d:\[Judas] Hamefura - S01E12.mkv"
[SUCCESS] Generated ed2k link.
[ERROR] Failed to get file info: 'utf-8' codec can't decode byte 0xe3 in position 1398: invalid continuation byte
```

--- a/anidbcli/encryptors.py
+++ b/anidbcli/encryptors.py
@@ -11,7 +11,7 @@ class PlainTextCrypto(TextCrypto):
     def Encrypt(self, message):
         return bytes(message, "utf-8")
     def Decrypt(self, message):
-        return message.decode("utf-8")
+        return message.decode("utf-8", errors="replace")
 
 BS = 16
 pad = lambda s: s + (BS - len(s) % BS) * chr(BS - len(s) % BS) 

